airflow/www/views.py

```diff
@@ -2144,7 +2146,7 @@ def trigger(self, dag_id: str, session: Session = NEW_SESSION):
                 form=form,
             )
 
-        dr = DagRun.find_duplicate(dag_id=dag_id, run_id=run_id, logical_date=logical_date)
+        dr = DagRun.find_duplicate(dag_id=dag_id, run_id=run_id)
```

Comment:
Same, it’d be a good idea to pass session in here.

Reply:
Changed it